_includes/community/community-meetings-table.html

```diff
@@ -0,0 +1,64 @@
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<div align="center"><h3 class="black-text">Layer5 Meetings</h3></div>
+
+<div style="overflow-x:auto;">
+<table id="table101">
+<tr>
+  <th>Group</th>
+  <th>Slack Channel</th>
+  <th>Meeting Minutes</th> 
+  <th>Meeting Link</th>
+  <th>Meeting Recordings</th>
+</tr>
+<tr>
+  <td><b><font color = #000000>Layer5 Community Meeting</font></b></td>
+  <td><a href = "https://layer5io.slack.com/archives/CDM0ACDM5">#general</a></td>
+  <td><a href="https://bit.ly/2It7IA4">Minutes</a></td>
+  <td><a href="https://meet.layer5.io/community">Zoom Link</a></td>
+  <td><a href = "https://www.youtube.com/playlist?list=PL3A-A6hPO2IMPPqVjuzgqNU5xwnFFn3n0">Youtube</a></td>
+</tr>
+<tr>
+  <td><b><font color = #000000>Websites Meeting</font></b></td>
+  <td><a href = "https://layer5io.slack.com/archives/C015QJKUMPU">#websites</a></td>
+  <td><a href = "https://bit.ly/2XK4eQV"> Minutes</a></td>
```

Comment:
Remove extraneous whitespace in " Minutes" hyperlink.

Remove whitespace between `href = "https..."`. Use `href="https..."`.